envi/qt/memcanvas.py

```diff
@@ -23,7 +23,7 @@
 class LoggerPage(QWebEnginePage):
     def javaScriptConsoleMessage(self, level, msg, line, source):
         print('------------------------------------------------------------------')
-        print(f'console message = {msg}')
+        print(f'console message = {msg}; line: {line}')
```

Comment:
py3.6 only

Reply:
It is, and we'll note that in the setup.